observations/r/__init__.py

```diff
@@ -0,0 +1,2308 @@
+"""[Observations.r](https://github.com/edwardlib/observations) provides
+a one line Python API for loading standard data sets in machine
+learning. It automates the process from downloading, extracting,
+loading, and preprocessing data. Observations helps keep the workflow
+reproducible and follow sensible standards.
+
+Observations is a standalone Python library and must be installed
+separate from Edward.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from observations.r.air_passengers import air_passengers
+from observations.r.bj_sales import bj_sales
+from observations.r.bod import bod
+from observations.r.co2 import co2
+from observations.r.formaldehyde import formaldehyde
+from observations.r.hair_eye_color import hair_eye_color
+from observations.r.insect_sprays import insect_sprays
+from observations.r.johnson_johnson import johnson_johnson
+from observations.r.lake_huron import lake_huron
+from observations.r.life_cycle_savings import life_cycle_savings
+from observations.r.nile import nile
+from observations.r.orchard_sprays import orchard_sprays
+from observations.r.plant_growth import plant_growth
+from observations.r.puromycin import puromycin
+from observations.r.titanic import titanic
+from observations.r.tooth_growth import tooth_growth
+from observations.r.ucb_admissions import ucb_admissions
+from observations.r.uk_driver_deaths import uk_driver_deaths
+from observations.r.uk_gas import uk_gas
+from observations.r.us_acc_deaths import us_acc_deaths
+from observations.r.us_arrests import us_arrests
+from observations.r.us_judge_ratings import us_judge_ratings
+from observations.r.us_personal_expend import us_personal_expend
+from observations.r.va_deaths import va_deaths
+from observations.r.www_usage import www_usage
+from observations.r.world_phones import world_phones
+from observations.r.airmiles import airmiles
```

Comment:
Is there a way to sort these alphabetically in your generation script? Same goes for writing to the `allowed_symbols` list.

Reply:
fixed